gap/projective/almostsimple.gi

```diff
@@ -1429,3 +1429,126 @@ FindHomMethodsProjective.SporadicsByOrders := function(ri,G)
   od;
   return false; # FIXME: false = NeverApplicable here really correct?
 end;
+
+#! @BeginChunk NameSporadic
+#! This method returns a list of sporadic simple groups that <A>ri</A>
+#! could be. It does not recognise the monster and the baby monster group. It
+#! is based on the magma v2.24.10 function <C>RecognizeSporadic</C>.
+FindHomMethodsProjective.NameSporadic := function(ri)
+    local orders, setOfOrders, maximalOrders, counter, namesOfPossibleSporadics, i, j;
+    orders := [];
+    #do we need SetPseudoRandomStamp?
+    for i in [1 .. 500] do
+        Add(orders, RandomElmOrd(ri, "NameSporadic", false).order);
+    od;
+    # Compute maximal orders. Maximal in the sense that it does not divide the
+    # order of another group element.
+    setOfOrders := Immutable(Set(orders));
```

Comment:
```suggestion
    setOfOrders := AsSet(orders);
```
First off, `Immutable` makes a copy of the set; you'd want `MakeImmutable` to avoid that. But best is to use `AsSet` which by definition directly returns an immutable set.